docs/containers-registries.conf.5.md

```diff
@@ -43,6 +43,8 @@ also include wildcarded subdomains in the format `*.example.com`.
 The wildcard should only be present at the beginning as shown in the formats
 above. Other cases will not work. For example, `*.example.com` is valid but
 `example.*.com`, `*.example.com/foo` and `*.example.com:5000/foo/bar:baz` are not.
+Note that `*` matches an arbitary number of subdomains. `*example.com` will hence
```

Comment:
@vrothberg one more update, please:
```suggestion
Note that `*` matches an arbitary number of subdomains. `*.example.com` will hence
```
the `.` is mandatory.

Reply:
https://github.com/containers/image/pull/1508